deploy/ansible/main.yml

```diff
@@ -1,13 +1,24 @@
 ---
-  - hosts: all
-    gather_facts: no
+- hosts: ubuntu
+  gather_facts: yes
     pre_tasks:
-
       - name: install python 2  # python installation (ansible hosts gather_facts requirement)
         raw: test -e /usr/bin/python || (apt -y update && apt install -y python)
         changed_when: False
 
-    roles:
-      - { role: iroha }
-      - { role: docker }
-      - { role: notary }
+- hosts: docker
+  gather_facts: yes
+  pre_tasks:
+    - name: install python 2
+      raw: tce-load -wi python
```

Comment:
Why do we use that `tce-load` instead of `apt` package manager? Is that some kind of a custom Linux distro that you assume `tce-load` is pre-installed on it?

Reply:
Yes, docker machine uses boot2docker, it does not have apt